src/components/tooltip/TooltipDirective.js

```diff
@@ -262,13 +263,15 @@ goog.require('ga_topic_service');
                 }, function(enabled) {
                   if (scope.isActive) {
                     if (enabled) {
-                      deregMapEvents();
+                      deregMapEvents = deregMapEvents();
                       deregGlobeEvents = registerGlobeEvents(scope,
                           findFeatures);
                     } else {
                       deregGlobeEvents();
-                      deregMapEvents = registerMapEvents(scope, findFeatures,
-                          gaBrowserSniffer.mobile);
+                      if (deregMapEvents === angular.noop) {
```

Comment:
I think we should be put this test at the begining of  `registerMapEvents`  and doiing the same in `registerGlobeEvents`